DocSum/docker/gaudi/docker_compose.yaml

```diff
@@ -71,6 +71,18 @@ services:
       - DOC_BASE_URL=${BACKEND_SERVICE_ENDPOINT}
     ipc: host
     restart: always
+  docsum-xeon-react-ui-server:
+    image: opea/docsum-react-ui:latest
+    container_name: docsum-gaudi-react-ui-server
+    depends_on:
+      - docsum-gaudi-backend-server
+    build:
```

Comment:
suggest to remove the build part.

Reply:
build part is need as we cannot inject environmental values to react once the image is built